### PR TITLE
Generate PDF for booking terms

### PR DESCRIPTION
**Pull request by matti-lamppu**
## 🛠️ Changelog
[//]: # "Describe the changes in this pull request here."

- Add endpoint for generating and downloading booking terms PDF
- Add option to admin panel for showing current PDF in browser
- Example file: [GENEROITU-Tilavarauspalvelu_yleiset_sopimusehdot.pdf](https://github.com/user-attachments/files/17011684/GENEROITU-Tilavarauspalvelu_yleiset_sopimusehdot.pdf) (original [here](https://github.com/City-of-Helsinki/tilavarauspalvelu-ui/blob/main/apps/ui/public/Tilavarauspalvelu_Yleiset_sopimusehdot.pdf))

## 🧪 Test plan
[//]: # "Help your fellow reviewer and write a short description of the fastest way to test your changes."

- Automated tests

## 🚧 Dependencies
[//]: # "Is this PR dependent on other changes outside this repository? Describe the changes, or add links to them."
[//]: # "e.g. This PR breaks X and is waiting for frontend changes before merging."

- None

## 🎫 Tickets
[//]: # "This pull request resolves all or part of the following ticket(s)."

- [TILA-3416](https://helsinkisolutionoffice.atlassian.net/browse/TILA-3416)

[TILA-3416]: https://helsinkisolutionoffice.atlassian.net/browse/TILA-3416?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: terms_of_use/templates/terms_of_use/components/base.jinja [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+
+    {% elif lang == 'sv' %}
+      {% set logo = logo_sv %}
+      {% set city_heading = 'HELSINGFORS STAD' %}
+      {% set government_heading = 'Centralförvaltningen' %}
+
+    {% else %}
+      {% set logo = logo_en %}
+      {% set city_heading = 'CITY OF HELSINKI' %}
+      {% set government_heading = 'Central government' %}
```
Comment (vergama) [inline terms_of_use/templates/terms_of_use/components/base.jinja:162]: Looks like the term which is used in EN translation is 'Central Administration'
Cf. e.g. https://www.hel.fi/en/decision-making/city-organisation/divisions

**Review by vergama — CHANGES_REQUESTED** [review]
Good to go, the only thing I noticed is the one translation.

**Review by vergama — APPROVED** [review]

**Review by ranta — APPROVED** [review]
Really clean implementation 🚀 

I didn't realise we didn't already have jinja installed 😱 I have been using its features already without issues 😅

**Comment by matti-lamppu** [comment]
@ranta We did have `jinja` installed, but not `django-jinja`. The latter includes some adapter logic for templating which is not available in django itself, and was required by the PDF converter.

**Pull request merged**